src/app/styles/terminal.sass

```diff
@@ -23,6 +22,9 @@
     .output
       font-size: 11px !important
 
+  iframe
+    padding: 10px
+
   .terminal
```

Comment:
everything below that line should be removed. please do not leave unused styles behind.